Gemfile

```diff
@@ -21,7 +21,7 @@ gem 'pry-nav', '~> 0.3.0'
 gem 'pry-stack_explorer', '~> 0.4.9' if RUBY_PLATFORM != 'java'
 gem 'rake', '>= 10.5'
 gem 'redcarpet', '~> 3.4' if RUBY_PLATFORM != 'java'
-gem 'rspec', '~> 3.0'
+gem 'rspec', '~> 3.10'
```

Comment:
The `without_partial_double_verification` API I make use of below was introduced in RSpec 3.6, but I decided to bump the minimum version to 3.10, since anyway it's still working with Ruby 2.0 so there's no gain in trying to support older RSpec versions.